crates/bevy_render/src/renderer/mod.rs

```diff
@@ -187,6 +187,14 @@ pub async fn initialize_renderer(
     let adapter_info = adapter.get_info();
     info!("{:?}", adapter_info);
 
+    if adapter_info.driver == "llvmpipe" || adapter_info.driver == "Microsoft Basic Display Adapter"
```

Comment:
Is there a reason this cannot just check whether `adapter_info.device_type == DeviceType::Cpu`?